to-do-list.js

```diff
@@ -0,0 +1,8 @@
+function createList(event) {
+  event.preventDefault();
```

Comment:
Do you need it here?

Reply:
I'm pretty sure this part has been commented out and not used in the final code so apparently it is not required here.